.env.example

```diff
@@ -1,2 +1,3 @@
 GITGUARDIAN_API_KEY=__________FILL ME__________
+TEST_GITGUARDIAN_API_KEY=__________FILL ME__________
```

Comment:
this is supposed to be copied by an user and not by a developer. 

I think it may be better to add it to the https://github.com/GitGuardian/ggshield/blob/main/CONTRIBUTING.md